php/libraries/NDB_BVL_Feedback.class.inc

```diff
@@ -369,7 +369,7 @@ class NDB_BVL_Feedback
         // new DB Object
         $db =& Database::singleton();
 
-        $query   = "SELECT Status FROM feedback_bvl_thread WHERE";
+        $query   = "SELECT MAX(Status) FROM feedback_bvl_thread WHERE";
```

Comment:
I would suggest to add ` 1=1 OR ` after the WHERE.

Reply:
There is no group by for the MAX function

Reply:
I'm not certain if any of those options are right (or the one that I have), the BVL Feedback integration test is still failing, but I'm not able to get bvl feedback to work on any branch, so I'm not sure why this PR is the one causing it to fail..